flutter_local_notifications_windows/test/xml_test.dart

```diff
@@ -56,25 +58,24 @@ const String complexXml = '''
 </toast>
 ''';
 
-void main() => group('XML', () {
-      FlutterLocalNotificationsWindows().enableMultithreading();
+void main() {
+  group('XML', () {
+    final FlutterLocalNotificationsWindows plugin =
+        FlutterLocalNotificationsWindows();
 
-      final FlutterLocalNotificationsWindows plugin =
-          FlutterLocalNotificationsWindows();
-      setUpAll(() => plugin.initialize(settings));
-      tearDownAll(() async {
-        await plugin.cancelAll();
-        plugin.dispose();
-      });
+    setUpAll(() => plugin.initialize(settings));
+    tearDownAll(() async {
+      await plugin.cancelAll();
+      plugin.dispose();
+    });
 
-      test('catches invalid XML', () async {
-        expect(plugin.showRawXml(id: 0, xml: emptyXml), throwsArgumentError);
-        expect(plugin.showRawXml(id: 1, xml: invalidXml), throwsArgumentError);
-        expect(
-            plugin.showRawXml(id: 2, xml: notWindowsXml), throwsArgumentError);
-        expect(
-            plugin.showRawXml(id: 3, xml: unmatchedXml), throwsArgumentError);
-        expect(plugin.showRawXml(id: 4, xml: validXml), completes);
-        expect(plugin.showRawXml(id: 5, xml: complexXml), completes);
-      });
+    test('catches invalid XML', () async {
+      expect(plugin.isValidXml(emptyXml), isFalse);
```

Comment:
Something I picked up after merging but these are all meant to be separate tests as there's a different permutations of input and output but have made as part of a single `test()`. The description given is also misleading as there are tests that assert valid XML was provided and is more appropriate as a group description. I noticed there are other test classes that are like this too

Reply:
Yes I plan to re-organize the tests, and have more thorough ones. For example, while there are tests to make sure the APIs can distinguish between valid and invalid XML, and tests to make sure the `WindowsNotificationsDetails` generates valid XML, there are no tests to map `WindowsNotificationDetails` to specific XML, eg, to verify that a progress bar is actually inserted. Another PR for another day :)